webapp/src/main/java/com/box/l10n/mojito/monitoring/QuartzMetricsReportingJobListener.java

```diff
@@ -0,0 +1,45 @@
+package com.box.l10n.mojito.monitoring;
+
+import org.quartz.JobExecutionContext;
+import org.quartz.JobExecutionException;
+import org.quartz.JobListener;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.stereotype.Component;
+
+import java.util.concurrent.TimeUnit;
+
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.Tags;
+
+@ConditionalOnProperty(value = "l10n.management.metrics.quartz-metrics.enabled", havingValue = "true")
```

Comment:
knit: `l10n.management.metrics.quartz-metrics.enabled` --> `l10n.management.metrics.quartz.enabled` , no need to repeat metric, no?